Cargo.toml

```diff
@@ -36,7 +36,7 @@ jsonwebtoken = { version = "9", default-features = false }
 log = "0.4.17"
 prost = "0.12"
 regorus = { version = "0.1.5", default-features = false, features = ["regex", "base64", "time"] }
-reqwest = { version = "0.12", default-features = false, features = ["default-tls"] }
+reqwest = { version = "0.12", default-features = false, features = ["default-tls", "blocking"] }
```

Comment:
I noticed that the `reqwest` is used in `fetch_vcek_from_kds`. It is suggested to use `reqwest::get` (the async version) rather than blocking version, because the blocking version would hang on network I/O during which the process would wait.

Reply:
I'm able to resolve most comments except this. Could we please have a conversation on slack?